tests/grpcio-no-wheel/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, poetry2nix, python3, runCommand }:
```

Comment:
Do we need the no-wheel test, since it's the default behaviour of poetry2nix to not install wheels?

What is being tested in this case? If there's anything special about grpcio, it would be nice to mention it at the top of this file.

Reply:
I think I was trying to be thorough here, but maybe it's better to just wait until there's an actual thing not working to add a test?